README.md

```diff
@@ -318,9 +322,11 @@ The expected command payload is :
 
  ```
  {
-    "cable": 32,
+    "cable": 32.0,
```

Comment:
Why do you modify the cable type?

Reply:
Because in the code, the update of the value is done only if cable is type float